samples/sampler/package.json

```diff
@@ -24,18 +24,21 @@
     "@enact/core": "^4.9.0",
```

Comment:
Should we bump up the version to 5.0.0-alpha.1?

Reply:
ok